src/panels/AssetsFolderPanel/AssetsFolderPanel.tsx

```diff
@@ -0,0 +1,103 @@
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { FilterList } from '@/components/FilterList/FilterList';
+import { Layout } from '@/components/Layout/Layout';
+import { LoadingBlocks } from '@/components/LoadingBlocks/LoadingBlocks';
+import { FolderBackIcon } from '@/icons/icons';
+import { IconSize } from '@/types/enums';
+
+import { AssetsBreadcrumbs } from './AssetsBreadcrumbs';
+import { AssetsEntry } from './AssetsEntry';
+import { FolderEntry } from './FolderEntry';
+import { useAssetFolders } from './hooks';
+import { Asset, AssetFolderNavigationState } from './types';
+import { collectAssets, findNavigatedFolder } from './util';
+
+export function AssetsFolderPanel() {
+  const rootFolder = useAssetFolders();
+  const [nav, setNav] = useState<AssetFolderNavigationState>();
+  const { t } = useTranslation('panel-assets');
+
+  useEffect(() => {
+    if (rootFolder) {
+      setNav({
+        root: rootFolder,
+        currentPath: []
+      });
+    }
+  }, [rootFolder]);
+
+  if (!nav) {
+    return <LoadingBlocks />;
+  }
+
+  const navigatedFolder = findNavigatedFolder(nav.root, nav.currentPath);
+  const nestedAssetsInCurrentFolder = collectAssets(navigatedFolder);
+
+  function wordInString(test: Asset, search: string): boolean {
+    return test.path.toLowerCase().includes(search.toLowerCase());
```

Comment:
Add a TODO comment about updating this?  It would be very useful to be able to narrow down the search using two words, for example.  

Or make an issue once this is merged :)